var/spack/repos/builtin.mock/packages/cray-mpich/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class CrayMpich(Package):
```

Comment:
Why was it necessary to add this package? Not clear from the description? Is there some test that strictly depend on it?

Reply:
Yes,  the three starting `test/package_class.py::test_possible_dependencies`.

Reply:
Going over the commits, it was initially needed for `test/cray_manifest.py` tests.

Reply:
This does not appear to be needed anymore -- nor the changes to `test/cmd/dependencies.py`, `test/cray_manifest.py`, or `test_package_class.py` when the builtin repository isn't available.

Note: The exception to this finding and others previously flagged is that, in general, there are still dependencies on `compiler-wrappers` and or `gcc-runtime` being symbolically linked to their builtin repository packages.